scene/animation/animation_tree_player.cpp

```diff
@@ -1314,6 +1314,7 @@ void AnimationTreePlayer::remove_node(const StringName &p_node) {
 		}
 	}
 
+	memdelete(node_map[p_node]);
```

Comment:
Is there a risk that the node could have been freed manually before this, which could lead to double-free?

Reply:
The actual pointer is not exposed to the outside of `AnimationTreePlayer`, users refer to a node by a string id when calling the API.